src/compactor.rs

```diff
@@ -247,6 +247,7 @@ impl CompactorOrchestrator {
             destination: compaction.destination,
             ssts,
             sorted_runs,
+            compaction_ts: db_state.last_clock_tick,
```

Comment:
ah, this is a gap in my knowledge. I assumed that the manifest is fetched whenever a compaction is run - if that's not the case then we should consider it. it's not wrong for the compaction job to run with an older timestamp, it just means it may not purge the most recently expired entries. Thoughts?

Reply:
Honestly, not sure. My understanding of the compaction is also a bit fuzzy. I'd have to look to see if the state is refreshed already. 😬